### PR TITLE
PREOPS-3286: Complete final version of document

### PR DESCRIPTION
**Pull request by DukeofSuffolk**

### CODE REVIEW

**Comment by leannep** [comment]
I'd suggest removing 2.2.14 NOIRLab Risk Tool -- do we want to expose the risk register via the documentation portal?

**Comment by leannep** [comment]
Do we want to expose Engineering Facility Database (EFD) information via the documentation portal? I wouldn't have thought so -- what is driving this?

File: StorageView.tex [hunk 1]
```diff
@@ -229,51 +188,76 @@ \subsubsection{GitHub}
...
+For all verification events performed by the SIT-COM team, a SIT-COM Test Plan/Report (SCTR) is created setting the standard for verification execution and results.
+The SCTR/DMTR is generated using the Jira REST API pulling the information from the verification objects.
+Formal detailed records of each verification done by Rubin Observatory is in the LSST Verification and Validation Jira project, with reports published to lsst.io as a SCTR or DMTR in addition to archiving it into DocuShare.
 
+Similar to MagicDraw, it is crucial that the normative source of information is clearly defined between the verification reports and other repositories which are synced or used for archival purposes, as well as the workflow and method to do so.
 
-\subsubsection{CMMS}
+\subsubsection{Computerized Maintenance Management System, CMMS}
 
+In operations, the project will use openMAINT\textregistered\ as a customize Computerized Maintenance Management System (CMMS), with the assistance from the managing company, Tecnoteca srl\copyright . \citep{openmaint-cite}
```
Comment (leannep) [inline StorageView.tex:244]:
```suggestion
In operations, the project will use openMAINT\textregistered\ as a customizable Computerized Maintenance Management System (CMMS), with the assistance from the managing company, Tecnoteca srl\copyright . \citep{openmaint-cite}
```

**Review by leannep — CHANGES_REQUESTED** [review]

**Comment by leannep** [comment]
In Section 2.1.1 Departments, Data Production needs to be updated to Data Management 
"Data Production (DP) — The role of the Rubin Observatory Data Production department is" -> Data management

**Review by leannep — APPROVED** [review]

**Review by cclaver — APPROVED** [review]
All changes are excellent and should be merged with the Master

**Pull request merged**